### DISCUSSION TITLE
No such host is known. (sts.us-west-2c.amazonaws.com:443)

### DISCUSSION THREAD

**Discussion by RyanXinno** [abuzuhri/Amazon-SP-API-CSharp#336] [General]
First of all thank you for the work.
I have struggled to get the order list.
Error Message shows "No such host is known. (sts.us-west-2c.amazonaws.com:443)"

It is my code below. Please let me know why I got an error.

ParameterOrderList serachOrderList = new ParameterOrderList();
serachOrderList.CreatedAfter = debugStartDate;
serachOrderList.OrderStatuses = new List<OrderStatuses>();
serachOrderList.OrderStatuses.Add(OrderStatuses.Shipped);
serachOrderList.MarketplaceIds = new List<string> { MarketPlace.US.ID };

try {
  var orders = amazonConnection.Orders.GetOrders(serachOrderList);
}
catch (Exception ex) {
  var Msg = ex.Message;  // ex.Message => "No such host is known. (sts.us-west-2c.amazonaws.com:443)"
}

**Comment by RyanXinno** [comment]
I used the CSharpAmazonSpAPI package version 1.4.30.

**Comment by abuzuhri** [comment]
I think its problem related to AWS setting 

Please look to confirmation file or use new AWS account for test and try again
